homeassistant/components/sensor/mold_indicator.py

```diff
@@ -174,7 +174,7 @@ def _calc_dewpoint(self):
             self._dewpoint = \
                 MAGNUS_K3 * (alpha + math.log(self._indoor_hum / 100.0)) / \
                 (beta - math.log(self._indoor_hum / 100.0))
-        _LOGGER.debug("Dewpoint: %f " + TEMP_CELSIUS, self._dewpoint)
+        _LOGGER.debug("Dewpoint: %f %s", self._dewpoint, TEMP_CELSIUS)
```

Comment:
`TEMP_CELSIUS` is a constant. However appending constants is essentially the same as appending variables in terms of speed (*except for potential compiler optimizations, though the speed impact is negligible)